ziti/src/main/kotlin/org/openziti/net/nio/AsyncSocketImpl.kt

```diff
@@ -205,13 +205,23 @@ internal class AsyncSocketImpl(private val connector: Connector = DefaultConnect
         if (optID == SocketOptions.SO_TIMEOUT) {
             return timeout.get()
         }
+
+        // Urgent data is not supported
+        if (optID == SocketOptions.SO_OOBINLINE) {
+            return false
+        }
         return null
     }
 
     override fun setOption(optID: Int, value: Any?) {
         if (optID == SocketOptions.SO_TIMEOUT) {
             timeout.set((value as Number).toInt())
         }
+
+        // Urgent data is not supported
+        if (optID == SocketOptions.SO_OOBINLINE) {
```

Comment:
should we throw exception on all unsupported options?

Reply:
We could.   We weren't before so I left it alone.   I'm happy to add an exception if you want me to.